examples/analog_hamiltonian_simulation/06_Analog_Hamiltonian_simulation_with_PennyLane.ipynb

```diff
@@ -22,7 +22,9 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.            if 'nan' in newkey:
</pre><p>This can be done prior to splitting the string. The else statement is not needed here as we pass on the key val pair.</p><p> </p><p>Additionally, we can simplify the for statement to:</p><p> </p><p><code>newkey = ''.join(['1' if a == '1.0' else '0' for a in newkey])</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>